edb/common/_typing_inspect.py

```diff
@@ -0,0 +1,118 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2011-present MagicStack Inc. and the EdgeDB authors.
+# Portions copyright 2017-2020 Ivan Levkivskyi
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I think `typing_inspect` is released under MIT, and so this file should as well.

Reply:
I'll fix